src/server/editorServices.ts

```diff
@@ -364,7 +365,7 @@ namespace ts.server {
         readonly openFiles: ScriptInfo[] = [];
 
         private compilerOptionsForInferredProjects: CompilerOptions;

```

Comment:
We would sometimes use both. `complierOptionsForInferredProjects` would be used for any workspace root that does not have its own distinct workspace settings in your _*.code-workspace_ file.